notebooks/ch12/linear_grad.md

```diff
@@ -166,6 +168,10 @@ y_vals = np.array([4, 5])
 assert np.allclose(grad_mse_cost(thetas, x_vals, y_vals), [0, 0])
 ```
 
+Like in Chapter 11, we use a `minimize` function that finds the optimal theta, but also accounts for our explanatory variable. It has the function signature (body omitted):
```

Comment:
Avoid ch number, that runs gradient descent, accounting for our explanatory variable.